client/state/index.js

```diff
@@ -57,6 +59,7 @@ export function createReduxStore( initialState, reducer = initialReducer ) {
 		isBrowser && require( './analytics/middleware.js' ).analyticsMiddleware,
 		isBrowser && require( './lib/middleware.js' ).default,
 		isAudioSupported && require( './audio/middleware.js' ).default,
+		isDesktop && require( '../../client/lib/desktop/middleware.js' ).default,
```

Comment:
Should this belong in the state folder at ./desktop/middleware.js

Reply:
Yep - that sounds more consistent. Moved.